src/data/scenarios_metadata_1.json

```diff
@@ -3,8 +3,8 @@
     "id": "scenario-001",
     "name": "ZETI Net Zero Pathway",
     "description": "A comprehensive pathway for the global energy sector to reach net zero by 2050",
-    "category": "IAM",
-    "category_tooltip": "Integrated Assessment Model",
+    "pathway_type": "Projection",
+    "pathway_type_tooltip": "Projection of future developments in sectors.",
```

Comment:
NB (and not strictly related to this PR): It seems _very_ brittle to define the `pathway_type_tooltip` manually, when it is uniquely determined by whatever the `pathway_type` is. I would prefer centrally managing a mapping from `pathway_type` to `pathway_type_tooltip` (likely, somewhere close to the `pbtar_schema`) and then programatically generating that

Reply:
Tracked in #235